### PR TITLE
Fix crash on error reporting on disallowed ref readonly returns

### LINKED ISSUE #19930: ref returning "this" in a "ref readonly" struct method crashes.
We hit unreachable code in the error reporting.

### PR DESCRIPTION
**Pull request by VSadov**
Fixes #19930

### CODE REVIEW

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - please review.

File: src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs [hunk 1]
```diff
@@ -1108,7 +1110,7 @@ private static ErrorCode GetMethodGroupLvalueError(BindValueKind valueKind)
 
         static private ErrorCode GetStandardLvalueError(BindValueKind kind)
         {
-            switch (kind)
+           switch (kind)
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs]: It looks like alignment is changed unintentionally.

Comment (VSadov) [inline src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs]: fixed

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - trivial fix. Anyone else wants to take a look?

**Comment by jcouv** [comment]
Looking

File: src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs [hunk 2]
```diff
@@ -1077,9 +1077,11 @@ private static ErrorCode GetRangeLvalueError(BindValueKind kind)
                 case BindValueKind.AddressOf:
                     return ErrorCode.ERR_InvalidAddrOp;
 
-                case BindValueKind.RefReturn:
-                case BindValueKind.ReturnableReference:
-                    return ErrorCode.ERR_RefReturnRangeVariable;
+            }
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs]: Nit: extra empty line above

**Comment by AlekseyTs** [comment]
LGTM

**Comment by VSadov** [comment]
test windows_debug_unit64_prtest please 
stuck for >hour in "git fetch"

**Pull request merged**